addons/account_check_printing/models/account_payment.py

```diff
@@ -96,8 +96,20 @@ def unmark_sent(self):
         self.write({'state': 'posted'})
 
     def do_print_checks(self):
-        """ This method is a hook for l10n_xx_check_printing modules to implement actual check printing capabilities """
-        raise UserError(_("You have to choose a check layout. For this, go in Apps, search for 'Checks layout' and install one."))
+        if not self:
+            raise _('Please select a record to print check.')
```

Comment:
How is this different from any other method that we put this raise?  (as other methods could have an empty self then too)

Reply:
@jco-odoo I can't recall why I did this but it looks like it is not needed, hence removing it.